sdccc/src/main/java/com/draeger/medical/sdccc/tests/glue/direct/ReportTestData.java

```diff
@@ -41,9 +42,10 @@ public ReportTestData(
         this.action = action;
         this.reportReceived = false;
         this.failOnReceivingReport = false;
+        this.subscriptionEndWithStatusDeliveryFailedReceived = false;
         this.syncPoint = new Object();
         this.subscribe = subscribe;
-        this.trigger = trigger;
+        this.trigger = trigger; // TODO: it would be simpler to pass the trigger as an override
```

Comment:
when is this TODO supposed to be done?

Reply:
then please create a GitHub issue for this, which can be marked as enhancement, and add its id in the source code here

Reply:
then please create a GitHub issue for this, which can be marked as enhancement, and add its id in the source code here

Reply:
Done. NOTE: The formatting looks it bit odd, but I'm afraind spotless is not open for negotiations.